packages/screens/LaunchpadERC20/component/LaunchpadERC20Banner.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Image, View } from "react-native";
+
+import LaunchpadERC20BannerImage from "../../../../assets/banners/launchpadERC20.jpg";
+import LogoSimpleSvg from "../../../../assets/icons/logo-simple.svg";
+
+import { BrandText } from "@/components/BrandText";
+import { SVG } from "@/components/SVG";
+import { SpacerColumn } from "@/components/spacer";
+import { useMaxResolution } from "@/hooks/useMaxResolution";
+import { useScreenSize } from "@/hooks/useScreenSize";
+import { fontSemibold16, fontSemibold28 } from "@/utils/style/fonts";
+import { ScreenSizeBreakpoints } from "@/utils/style/layout";
+
+export const LaunchpadERC20Banner: React.FC = () => {
+  const { width } = useMaxResolution();
+  const isSmallScreen = useScreenSize() === ScreenSizeBreakpoints.SMALL;
```

Comment:
Please do the same as my suggestion on [LaunchpadBanner.tsx](https://github.com/TERITORI/teritori-dapp/pull/1276/files#diff-3fae2aa36d28d540e0cbc4783088d732f1bd39491fbcc6d46ea827d4b8a4b2a7) if you are ok

Reply:
[b55404e](https://github.com/TERITORI/teritori-dapp/pull/1276/commits/b55404ed123b48ed783f20ea4af3e3c08881b8f4)